sdk/core/core-http/review/core-http.api.md

```diff
@@ -953,6 +966,12 @@ export interface WebResourceLike {
     withCredentials: boolean;
 }
 
+// @public
+export const XML_ATTRKEY = "$";
+
+// @public
+export const XML_CHARKEY = "_";
```

Comment:
I feel it is useful to export the default value, since we now support overriding it.